symphony-bdk-core/src/test/java/com/symphony/bdk/core/config/BdkConfigParserTest.java

```diff
@@ -11,112 +11,166 @@
-        String configPath = "/config/config_properties.json";
-        InputStream inputStream = BdkConfigLoaderTest.class.getResourceAsStream(configPath);
-        JsonNode jsonNode = BdkConfigParser.parse(inputStream);
-        assertEquals("${my.property}/privatekey.pem", jsonNode.at("/bot/privateKeyPath").asText());
-    }
-
-    @Test
-    void parseYamlConfigTest() throws BdkConfigException {
-        String configPath = "/config/config.yaml";
-        InputStream inputStream = BdkConfigLoaderTest.class.getResourceAsStream(configPath);
-        JsonNode jsonNode = BdkConfigParser.parse(inputStream);
-        assertEquals("tibot", jsonNode.at("/bot/username").asText());
-    }
-
-    @Test
-    void parseYamlConfigWithPropertyTest() throws BdkConfigException {
-        System.setProperty("my.property", "propvalue");
-        String configPath = "/config/config_properties.yaml";
-        InputStream inputStream = BdkConfigLoaderTest.class.getResourceAsStream(configPath);
-        JsonNode jsonNode = BdkConfigParser.parse(inputStream);
-        assertEquals("propvalue/privatekey.pem", jsonNode.at("/bot/privateKey/path").asText());
-    }
-
-    @Test
-    void parseYamlConfigWithPropertiesInArray() throws BdkConfigException {
-        System.setProperty("my.property", "agent-lb.acme.org");
-        String configPath = "/config/config_lb_properties.yaml";
-        InputStream inputStream = BdkConfigLoaderTest.class.getResourceAsStream(configPath);
-        final JsonNode loadBalancing = BdkConfigParser.parse(inputStream).at("/agent/loadBalancing");
-
-        assertEquals("roundRobin", loadBalancing.at("/mode").asText());
-        assertTrue(loadBalancing.at("/stickiness").asBoolean());
-
-        assertEquals("agent1.acme.org", loadBalancing.at("/nodes/0/host").asText());
-        assertEquals(1234, loadBalancing.at("/nodes/0/port").asInt());
-
-        assertEquals("agent-lb.acme.org", loadBalancing.at("/nodes/1/host").asText());
-    }
-
-    @Test
-    void parseInvalidConfigTest() {
-        String configPath = "/config/invalid_config.html";
-        BdkConfigException exception = assertThrows(BdkConfigException.class, () -> {
-            InputStream inputStream = BdkConfigLoaderTest.class.getResourceAsStream(configPath);
-            BdkConfigParser.parse(inputStream);
-        });
-        assertEquals("Given InputStream is not valid. Only YAML or JSON are allowed.", exception.getMessage());
-    }
-
-    @Test
-    void parseInvalidYamlConfigTest() {
-        String configPath = "/config/invalid_config.yaml";
-        BdkConfigException exception = assertThrows(BdkConfigException.class, () -> {
-            InputStream inputStream = BdkConfigLoaderTest.class.getResourceAsStream(configPath);
-            BdkConfigParser.parse(inputStream);
-        });
-        assertEquals("Given InputStream is not valid. Only YAML or JSON are allowed.", exception.getMessage());
+  }
+
+  @BeforeEach
+  void tearDown() {
```

Comment:
`tearDown` for a `@BeforeEach` method is a bit misleading IMHO. `setUp` is a more usual name for that kind of method.

Reply:
I agree!

https://github.com/finos/symphony-bdk-java/commit/458eb2687ee4186651aba8093ee7016884ae2a4d#diff-770895de718032b550984a565ad529dae57cb73e6d7288826459ee1c09248528